### DISCUSSION TITLE
Cost_cap_fin_mult related questions

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#130] [Q&A: Analysis]
Hi NREL team,

I am confused about the meaning of including 3-6 parts of the financial multipliers in cost_cap_fin_mult (screenshot attached).

What is the result of using 3-6 financial parameters multiplied by the overnight cost of capital in OF? Does it try to calculate the revenue requirement for any given plant?

Thanks! Attached is a screen shot.

Cost_cap_fin_mult (unitless) = Regional_cost_multiplier (fraction) * Financial Multipliers (unitless)
and
Financial Multipliers (unitless) equation shows
![image](https://user-images.githubusercontent.com/78582696/189387461-db0ca962-cb4e-4b6f-9e19-b4dd1a3311c1.png)

**Comment by wesleyjcole** [comment]
Parts 3-6 of the equation above adjust other incentives to be pre-tax values.  Parts 1 and 2 do not include the effects of decreciation or tax credits, so those are factored in vial parts 3-6.  Because these incentives are not taxed, we inflate their values by (1 - TaxRate) to make sure they are properly valued.

**Reply by wanyingw193** [reply]
Thank you for your reply. I can't fully understand the last sentence. Would you mind explaining more about "these incentives are not taxed"?

Looked at the source of the financial equations (W. Short, Packey, and Holt 1995) and found that parts 3-6 are similar to equations (2-22) and (4-4). Does the parts 3-6 formula derive from one of them? Thanks!
(2-22):
![image](https://user-images.githubusercontent.com/78582696/189953984-004af0f4-493e-4e1a-b4a8-396a87225710.png)
(4-4):
![image](https://user-images.githubusercontent.com/78582696/189954290-9640f38f-4705-43a9-8058-1f122f35d44e.png)

**Reply by wesleyjcole** [reply]
Let's say you receive $1,000 from the government as a tax credit.  You don't have to pay taxes on that $1,000.  If your tax rate is 25%, then that $1,000 is equivalent to 1000 / (1-0.25) = $1,333 of taxable income.
